mods/voidmulti.sc2mod/base.sc2data/gamedata/behaviordata.xml

```diff
@@ -33,7 +42,16 @@
         <Start value="Footprint4x4CreepSourceGrown"/>
         <Birth value="CreepGrowthMediumNydusCreepSourceStart"/>
         <Grown value="Footprint4x4CreepSourceGrown"/>
+        <InfoFlags index="Hidden" value="1"/>
```

Comment:
**undocumented**: Nydus Worms do not display a buff/debuff icon anymore when selected